examples/README.md

```diff
@@ -0,0 +1,61 @@
+# Running Examples
+
+Run the examples in this directory with:
+```sh
+# Ensure you are in the examples directory
+cd examples/
+
+# Run example
+python3 <example>.py
```

Comment:
```suggestion
# Run example
python3 examples/<example>.py
```

Reply:
I'm not sure I like having all the examples in the same directory. Subdirectories are self contained. Examples like pull and create and provide additional information to the user without it bleeding into other examples

Reply:
I gave this some thought too and tried both things. While you're right individual directories would cause less bleeding into each other - it's just easier to run and play around with examples without having to jump between directories.
I can imagine newer users go into each directory to run the example lol.